src/firefly/js/visualize/WebPlot.js

```diff
@@ -311,6 +316,28 @@ function makePlotTemplate(plotId, plotType, asOverlay, imageCoordSys) {
 }
 
 
+function processAllAltWcs(header) {
+
+    const availableAry= alphabetAry.filter( (c) => header['CTYPE1'+c]);
+    if (isEmpty(availableAry)) return {};
+
+    return availableAry.reduce( (obj, altChar) => {
+        const processHeader= parseSpacialHeaderInfo(header, altChar);
```

Comment:
is 'parseSpacialHeaderInfo' for both spacial WCS and alternative WCS axis description?